integration_tests/e2e/applications.cy.ts

```diff
@@ -0,0 +1,30 @@
+import ApplicationListPage from '../pages/applicationsList'
+
+import applicationSummaryFactory from '../../server/testutils/factories/applicationSummary'
+
+context('Applications dashboard', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+  })
+
+  it('shows the dashboard ', () => {
+    // Given I am logged in
+    cy.signIn()
+
+    // And there are applications in the database
+    const applicationSummaries = applicationSummaryFactory.buildList(5)
+    cy.task('stubApplications', applicationSummaries)
+
+    // When I visit the Previous Applications page
+    const page = ApplicationListPage.visit()
+
+    // Then I should see all of the application summaries listed
+    page.shouldShowApplicationSummaries(applicationSummaries)
+
+    // And I the button should take me to the application start page
+    page.clickSubmit()
+    cy.get('h1').should('contain', 'Apply for an Approved Premises (AP) placement')
```

Comment:
This could probably use the `StartPage` page class I add in #157 (Depends on what gets merged first)

Reply:
I clocked this on your PR - if you merge yours first I'll update this. Otherwise I may forget :D